src/main/java/com/microsoft/cse/helium/app/config/CosmosDbConfig.java

```diff
@@ -0,0 +1,52 @@
+package com.microsoft.cse.helium.app.config;
+
+import com.microsoft.cse.helium.app.Constants;
+import com.microsoft.cse.helium.app.services.configuration.IConfigurationService;
+
+import com.azure.data.cosmos.ConsistencyLevel;
+import com.azure.data.cosmos.ConnectionPolicy;
+import com.azure.data.cosmos.RetryOptions;
+import com.azure.data.cosmos.internal.RequestOptions;
+import com.microsoft.azure.spring.data.cosmosdb.config.AbstractCosmosConfiguration;
+import com.microsoft.azure.spring.data.cosmosdb.config.CosmosDBConfig;
+import com.microsoft.azure.spring.data.cosmosdb.repository.config.EnableCosmosRepositories;
+import com.microsoft.azure.spring.data.cosmosdb.repository.config.EnableReactiveCosmosRepositories;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableCosmosRepositories(basePackages = "com.microsoft.azure.helium.app.*")
+public class CosmosDbConfig extends AbstractCosmosConfiguration {
+
+    protected IConfigurationService _configService;
+    protected String _cosmosContainer= "";
+    protected String _cosmosDatabase= "";
+    protected final RequestOptions _requestOptions = new RequestOptions();
+
+    @Autowired
+    public CosmosDbConfig(IConfigurationService configService){
+        _configService = configService;
+        _cosmosContainer = _configService.getConfigEntries().get(Constants.COSMOS_COLLECTION_KEYNAME);
+        _cosmosDatabase = _configService.getConfigEntries().get(Constants.COSMOS_DATABASE_KEYNAME);
+
+        _requestOptions.setConsistencyLevel(ConsistencyLevel.CONSISTENT_PREFIX);
+        _requestOptions.setScriptLoggingEnabled(true);
+    }
+
+    @Bean
+    @Primary
+    public CosmosDBConfig BuildCosmosDBConfig() {
```

Comment:
we can follow standard camel case nomenclature buildCosmosDBConfig

Reply:
Done.